redfish-core/lib/update_service.hpp

```diff
@@ -1065,6 +1066,10 @@ inline void requestRoutesSoftwareInventory(App& app)
                 sw_util::getSwStatus(asyncResp, swId, obj.second[0].first);
                 getSoftwareVersion(asyncResp, obj.second[0].first, obj.first,
                                    *swId);
+                asyncResp->res.jsonValue["Name"] = "Software Inventory";
+
+                name_util::getPrettyName(asyncResp, obj.first, obj.second,
```

Comment:
We don't want this here: https://github.com/ibm-openbmc/bmcweb/commit/2ff339a970456d637836b5202d8b13cf1a06d0b9